tests/helper/test-case.js

```diff
@@ -30,9 +30,6 @@ var TokenStreamTransformer = require("./token-stream-transformer");
  * If the file contains more than three parts, the remaining parts are just ignored.
  * If the file however does not contain at least two parts (so no expected token stream),
  * the test case will later be marked as failed.
- *
- *
- * @type {{runTestCase: Function, transformCompiledTokenStream: Function, parseTestCaseFile: Function}}
```

Comment:
I don't think this should be removed?

Reply:
It should because the typing is incomplete and only gets in the way of TS's type inference and the doc/types provided by the JSDoc of the functions.